src/ui/public/vis/editors/default/default.js

```diff
@@ -74,7 +75,9 @@ const defaultEditor = function ($rootScope, $compile) {
           }
 
           $scope.submitEditorWithKeyboard = (event) => {
-            if (event.ctrlKey && event.keyCode === 13) {
+            event.preventDefault();
```

Comment:
Only cancel the event, if it was the ctrl + enter event, should be inside the if (same for stopPropagation)